babel.dependencies.config.js

```diff
@@ -13,6 +13,7 @@ const config = {
 				exclude: [ 'transform-typeof-symbol' ],
 			},
 		],
+		'@babel/preset-typescript',
```

Comment:
I wouldn't add the preset to the `babel.dependencies.config.js` unless it's actually needed. That will happen once some NPM package gets published with TypeScript sources only.